code/modules/projectiles/special.dm

```diff
@@ -1138,3 +1138,46 @@ ABSTRACT_TYPE(/datum/projectile/special)
 		var/turf/T = get_turf(O)
 		src.emit_chems(T, O)
 		src.emit_gas(T, 1)
+
+
+/datum/projectile/special/slug_slime	//Goos people and doors!
+	name = "slime"
+	dissipation_rate = 1
+	dissipation_delay = 7
+	icon_state = "slime_shot"
+	power = 5
+	hit_ground_chance = 0
+	ks_ratio = 1.0
```

Comment:
define projectiles with `damage` and `stun`. `power` and `ks_ratio` are autogenerated from those values